content/updates/2020-11/1-Stop-your-imports-and-exports.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_alt_img: "The new Stop button in the process tracker"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Stop your imports and exports
```

Comment:
```suggestion
# Easily stop ongoing jobs
```
It's not only your jobs, it's any user job if you have the permission, and since it's not only imports and exports, I'd rather have "jobs" in the title.